projects/1.18/assets/macaws-roofs-oh-the-biomes-youll-go/macawsroofsbyg/lang/zh_cn.json

```
@@ -0,0 +1,452 @@
+	"block.macawsroofsbyg.cika_upper_lower_roof":"锡卡木平缓屋顶顶端",
+	"block.macawsroofsbyg.cika_upper_steep_roof":"锡卡木陡峭屋顶底端",
+	"block.macawsroofsbyg.cika_planks_roof":"锡卡木木板屋顶",
+	"block.macawsroofsbyg.cika_planks_attic_roof":"锡卡木木板天窗屋顶",
+	"block.macawsroofsbyg.cika_planks_cube_roof":"锡卡木木板方块屋顶",
+	"block.macawsroofsbyg.cika_planks_top_roof":"锡卡木木板屋顶顶端",
+	"block.macawsroofsbyg.cika_planks_lower_roof":"锡卡木木板平缓屋顶",
+	"block.macawsroofsbyg.cika_planks_steep_roof":"锡卡木木板陡峭屋顶底端",
+	"block.macawsroofsbyg.cika_planks_upper_lower_roof":"锡卡木木板平缓屋顶顶端",
+	"block.macawsroofsbyg.cika_planks_upper_steep_roof":"锡卡木木板陡峭屋顶顶端",
+	"block.macawsroofsbyg.cypress_roof":"柏木屋顶",
+	"block.macawsroofsbyg.cypress_attic_roof":"柏木天窗屋顶",
+	"block.macawsroofsbyg.cypress_cube_roof":"柏木方块屋顶",
+	"block.macawsroofsbyg.cypress_top_roof":"柏木屋顶顶端",
+	"block.macawsroofsbyg.cypress_lower_roof":"柏木平缓屋顶底端",
+	"block.macawsroofsbyg.cypress_steep_roof":"柏木陡峭屋顶",
+	"block.macawsroofsbyg.cypress_upper_lower_roof":"柏木顶端平缓屋顶",
+	"block.macawsroofsbyg.cypress_upper_steep_roof":"柏木顶端陡峭屋顶底端",
+	"block.macawsroofsbyg.cypress_planks_roof":"柏木木板屋顶",
+	"block.macawsroofsbyg.cypress_planks_attic_roof":"柏木木板天窗屋顶",
+	"block.macawsroofsbyg.cypress_planks_cube_roof":"柏木木板方块屋顶",
+	"block.macawsroofsbyg.cypress_planks_top_roof":"柏木木板屋顶顶端",
+	"block.macawsroofsbyg.cypress_planks_lower_roof":"柏木木板平缓屋顶",
+	"block.macawsroofsbyg.cypress_planks_steep_roof":"柏木木板陡峭屋顶底端",
+	"block.macawsroofsbyg.cypress_planks_upper_lower_roof":"柏木木板平缓屋顶顶端",
+	"block.macawsroofsbyg.cypress_planks_upper_steep_roof":"柏木木板陡峭屋顶顶端",
+	"block.macawsroofsbyg.ebony_roof":"乌木屋顶",
+	"block.macawsroofsbyg.ebony_attic_roof":"乌木天窗屋顶",
+	"block.macawsroofsbyg.ebony_cube_roof":"乌木方块屋顶",
+	"block.macawsroofsbyg.ebony_top_roof":"乌木屋顶顶端",
+	"block.macawsroofsbyg.ebony_lower_roof":"乌木平缓屋顶底端",
+	"block.macawsroofsbyg.ebony_steep_roof":"乌木陡峭屋顶",
+	"block.macawsroofsbyg.ebony_upper_lower_roof":"乌木顶端平缓屋顶",
+	"block.macawsroofsbyg.ebony_upper_steep_roof":"乌木顶端陡峭屋顶底端",
+	"block.macawsroofsbyg.ebony_planks_roof":"乌木木板屋顶",
+	"block.macawsroofsbyg.ebony_planks_attic_roof":"乌木木板天窗屋顶",
+	"block.macawsroofsbyg.ebony_planks_cube_roof":"乌木木板方块屋顶",
+	"block.macawsroofsbyg.ebony_planks_top_roof":"乌木木板屋顶顶端",
+	"block.macawsroofsbyg.ebony_planks_lower_roof":"乌木木板平缓屋顶",
+	"block.macawsroofsbyg.ebony_planks_steep_roof":"乌木木板陡峭屋顶底端",
+	"block.macawsroofsbyg.ebony_planks_upper_lower_roof":"乌木木板平缓屋顶顶端",
+	"block.macawsroofsbyg.ebony_planks_upper_steep_roof":"乌木木板陡峭屋顶顶端",
+	"block.macawsroofsbyg.ether_roof":"以太木屋顶",
+	"block.macawsroofsbyg.ether_attic_roof":"以太木天窗屋顶",
+	"block.macawsroofsbyg.ether_cube_roof":"以太木方块屋顶",
+	"block.macawsroofsbyg.ether_top_roof":"以太木屋顶顶端",
+	"block.macawsroofsbyg.ether_lower_roof":"以太木平缓屋顶底端",
+	"block.macawsroofsbyg.ether_steep_roof":"以太木陡峭屋顶",
+	"block.macawsroofsbyg.ether_upper_lower_roof":"以太木顶端平缓屋顶",
+	"block.macawsroofsbyg.ether_upper_steep_roof":"以太木顶端陡峭屋顶底端",
+	"block.macawsroofsbyg.ether_planks_roof":"以太木板屋顶",

```

```suggestion
	"block.macawsroofsbyg.ether_planks_roof":"以太木木板屋顶",
```
下同
不过这样确实叫起来不大顺口
但就这样吧（摆烂）